frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,179 @@
+  },
+
+  switchTo : (perspectiveName: switchPerspective) => {
+    switch (perspectiveName) {
+      case switchPerspective.Administrator: {
+        nav.sidenav.switcher.changePerspectiveTo('Administrator');
+        break;
+      }
+      case switchPerspective.Developer: {
+        nav.sidenav.switcher.changePerspectiveTo('Developer');
+        break;
+      }
+      default: {
+        throw new Error('Option is not available');
+      }
+  }
+}
+};
+
+export const naviagteTo = (opt: devNavigationMenu) => {
+  switch (opt) {
+    case devNavigationMenu.Add: {
+      cy.byLegacyTestID('+Add-header').click().then(() => {
+        cy.url().should('include', 'add');
+        app.waitForLoad();
+      });
+      break;
+    }
+    case devNavigationMenu.Topology: {
+      cy.byLegacyTestID('topology-header').click()
+      cy.url().should('include', 'topology');
+      break;
+    }
+    case devNavigationMenu.GitOps: {
+      cy.byLegacyTestID('gitops-header').click();
+      detailsPage.titleShouldContain('GitOps');
+      break;
+    }
+    case devNavigationMenu.Monitoring: {
+      cy.byLegacyTestID('monitoring-header').click();
+      detailsPage.titleShouldContain('Monitoring');
+      break;
+    }
+    case devNavigationMenu.Builds: {
+      cy.byLegacyTestID('build-header').click();
+      detailsPage.titleShouldContain('Build Configs');
+      break;
+    }
+    case devNavigationMenu.Pipelines: {
+      cy.byLegacyTestID('pipeline-header').click();
+      detailsPage.titleShouldContain('Pipelines');
+      break;
+    }
+    case devNavigationMenu.Search: {
+      cy.byLegacyTestID('search-header').click();
+      detailsPage.titleShouldContain('Search');
+      break;
+    }
+    case devNavigationMenu.Helm: {
+      cy.byLegacyTestID('helm-releases-header').click();
+      detailsPage.titleShouldContain('Helm Releases');
+      break;
+    }
+    case devNavigationMenu.Project: {
+      cy.byLegacyTestID('project-details-header').click();
+      break;
+    }
+    case devNavigationMenu.ConfigMaps: {
+      cy.get('#ConfigMap').click();
+      detailsPage.titleShouldContain('Config Maps');
+      break;
+    }
+    case devNavigationMenu.Secrets: {
+      cy.get('#Secret').click();
+      detailsPage.titleShouldContain('Secrets');
+      break;
+    }
+    default: {
+      throw new Error('Option is not available');
+    }
+  }
+};
+
+export const projectNameSpace = {
+  selectCreateProjectOption: () => {
+    cy.byLegacyTestID('namespace-bar-dropdown')
+      .find('button')
+      .eq(0)
+      .click();
+    cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
+  },
+
+  enterProjectName: (projectName: string) => {
+    cy.get('form[name="form"]').should('be.visible');
+    cy.get('#input-name').type(projectName);
+  },
+
+  clickCreateButton: () => {
+    cy.get('#confirm-action').click();
+  },
+
+  createNewProject: (projectName: string) => {
```

Comment:
There is a small change in between these functions, 
In dev console ui -> 
 cy.byLegacyTestID('resource-title').should('have.text', name); this step will be failed

Reply:
Hi, why would this fail?  Will the `resource-title` not have the `name`?  If there is a difference between admin and dev-console, then I would recommend updating `/integration-tests-cypress/support/project.ts` `createProject` to take a parameter like `skipTitleCheck` or `inDevConsole: false` (not sure exactly how to word it as I'm not familar with the difference).
The main point is we want all devs to use and enhance the same set of helper methods.  This is, unfortunately, what happened with the Protractor tests, everyone wrote the same helper methods over and over again, in different files, and it mushroomed out of control.

Reply:
resource-title contains the page name where the user is currently present
Eg: If the user is on Installed Operators page and tried to create the project then the resource-title displays as "Installed Operators" not project name
![image](https://user-images.githubusercontent.com/61005404/94696156-5994eb00-0354-11eb-91a3-593b3e2e69a4.png)

Reply:
Thanks for the explanation :-)  I recommend updating `/integration-tests-cypress/support/project.ts` `createProject` to take a parameter `devConsole: false`, where if `devConsole === true` then `createProject` will check for resource-title to be `Installed Operators` instead of the project name....or whatever naming/logic you feel more closly resembles where the test is running.

Reply:
:+1: like the changes.  Just one question, in [createProject()](https://github.com/openshift/console/pull/6776/commits/cf66d4d1c588adcf0ab243546f9099cf696a199e#diff-d7b862ece419a8b80b099f7dd4998539R27)  should we test for "Installed Operators" if devConsole === true?

Reply:
In dev-console, after creating project, we are nor verifying the title of the page. as it always changes.

Reply:
ok